Formula/xrootd.rb

```diff
@@ -23,8 +23,13 @@ class Xrootd < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "libcython" => :build
+  depends_on "python@3.10" => [:build, :test]
```

Comment:
Why not the newest?

Reply:
I saw that Python 3.10 was being used when trying to build from source, which caused failures. XRootD in principle can work fine with 3.11.

Reply:
Same goes for OpenSSL 3.x. It should work, but there were problems with the dependency tree, so I went back to 1.1.

Reply:
This is what I see when trying to build (in docker) with Python 3.11
```
CMake Warning at bindings/python/CMakeLists.txt:55 (message):
   /home/linuxbrew/.linuxbrew/Cellar/xrootd/5.5.2/bin/python3.11 does not have a valid pip associated with it. It is recommended that you install a version of pip to install Python packages and bindings. If you are unable to install a version of pip through a package manager or with your Python build try using the PyPA's get-pip.py bootstrapping script ( https://github.com/pypa/get-pip ).
   The installation of the Python bindings will attempt to continue using the deprecated method of `/home/linuxbrew/.linuxbrew/Cellar/xrootd/5.5.2/bin/python3.11 setup.py install`.
```
I tried `brew reinstall python@3.11` and `python3.11 -m pip install --upgrade pip`, but still got this error, so I think for this time it's better to stick with 3.10.

Reply:
Alright, I think I learned how to use both Python3.11 and OpenSSL@3. The new formula version is using them.